src/main/java/edu/ucsb/cs156/happiercows/repositories/ProfitRepository.java

```diff
@@ -0,0 +1,12 @@
+package edu.ucsb.cs156.happiercows.repositories;
+
+import edu.ucsb.cs156.happiercows.entities.Profit;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ProfitRepository extends CrudRepository<Profit, Long> {
+  Iterable<Profit> findAllByUserCommonsId(Long user_commons_id);
+
+  Iterable<Profit> findAll();
```

Comment:
This comes for free with `CrudRepository`, so we could probably remove this.

Reply:
Thanks for letting me know! It has been removed.